src/erlmld_app.erl

```diff
@@ -15,11 +15,11 @@ start(_StartType, []) ->
 stop(_State) ->
     ok.
 
-ensure_all_started() ->
+ensure_all_deps_started() ->
     {ok, Deps} = application:get_key(erlmld, applications),
-    {ok,
-     lists:append([begin
-                       {ok, Started} = application:ensure_all_started(Dep),
-                       Started
-                   end
-                   || Dep <- Deps])}.
+    lists:foreach(fun(Dep) ->
+                     {ok, Started} = application:ensure_all_started(Dep),
```

Comment:
Your can just do `{ok,_}` since you are using `lists:foreach/2` anyway.

Reply:
Yep, that's true, fixed